packages/plugin-ext/src/main/browser/view/tree-views-main.tsx

```diff
@@ -126,12 +126,15 @@ export class TreeViewsMainImpl implements TreeViewsMain {
             this.proxy.$setExpanded(treeViewId, event.id, event.expanded);
         });
 
+        treeViewWidget.onSelectionChanged(event => {
```

Comment:
Why do you handle selection event twice?

Reply:
It's two different events. Couldn't find better name for this thing.